tests/ci/docs_check.py

```diff
@@ -84,7 +93,7 @@ def main():
     elif args.force:
         logging.info("Check the docs because of force flag")
 
-    docker_image = get_image_with_version(reports_path, "clickhouse/docs-builder")
+    docker_image = pull_image(get_docker_image("clickhouse/docs-builder"))
```

Comment:
This is used in https://github.com/ClickHouse/clickhouse-docs/blob/main/.github/workflows/pull-request.yml#L44

Will it work now?

Reply:
https://github.com/ClickHouse/ClickHouse/pull/56317/files#diff-1343081d14003c3b9bf79e16c29bbd6e8c2be4b0509a37d5faa36b3eeb98cf4dR65

I think, we thought about the fallback to the `latest`